snapcraft/storeapi/__init__.py

```diff
@@ -91,6 +92,10 @@ def __init__(self, conf, root_url):
         self.conf = conf
         self.root_url = root_url
         self.session = requests.Session()
+        # Setup max retries for all store URLs and the CDN
+        self.session.mount('http://', HTTPAdapter(max_retries=5))
+        self.session.mount('https://', HTTPAdapter(max_retries=5))
```

Comment:
This looks magical! Although I want to point out: I often see multiple PRs fail at the same time, which seems like the CDN goes down completely for a period of time. Any idea how quickly these retries occur?